tests/testthat/test-combine-categories.R

```diff
@@ -123,6 +123,27 @@ with_mock_crunch({
             list(list(name="Both", categories=NULL))),
             "Combinations must reference 'categories' by name or id")
     })
+
+    test_that("collapseCategories errors correctly", {
+        expect_error(collapseCategories(ds$birthyr, "from", "to"),
+            "Variable must be a categorical.")
+        expect_error(collapseCategories(ds$gender, "Male", 1),
+            "Destination category must be a character string of length 1.")
+        expect_error(collapseCategories(ds$gender, "Male", c("young", "old")),
+            "Destination category must be a character string of length 1.")
+        expect_error(collapseCategories(ds$gender, 1, "young"),
+            paste0(sQuote('from'), " must be a character vector or logical expression."))
```

Comment:
I think `dQuote` is the standard in the package (there are 77 instances of `dQuote` and 7 of `sQuote`) so let's use that.